doc/cephfs/quota.rst

```diff
@@ -69,9 +69,12 @@ If the attributes appear on a directory inode that means a quota is
 configured there.  If they are not present then no quota is set on
 that directory (although one may still be configured on a parent directory).
 
+The value of the ``ceph.quota.max_bytes`` must be aligned to 4MB if it's
```

Comment:
Reference tracker id is missing in this commit.

Reply:
Added it. Thanks.

Reply:
s/of the/of/